### PR TITLE
add some enums and tooling to generate it

### PR DESCRIPTION
**Pull request by ferrangrau**
@joaoqalves can you review this?

### CODE REVIEW

File: tooling/generate_instances_types.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import boto3
+
+session = boto3.Session(region_name="us-east-1", profile_name="sch-gov")
```
Comment (joaoqalves) [inline tooling/generate_instances_types.py]: Profile name should be passed

File: tooling/generate_instances_types.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+pricing = session.client('pricing')
+
+
+response = pricing.get_attribute_values(ServiceCode='AmazonEC2', AttributeName='instanceType')
+
+
+path = 'type.py'
```
Comment (joaoqalves) [inline tooling/generate_instances_types.py]: When will we generate this? Build time?

File: tooling/generate_instances_types.py (hunk 1 above)
Comment (ferrangrau) [reply tooling/generate_instances_types.py]: Sorry @joaoqalves I can do it better, it's only a quick tool to help me to generate the instances type enum...

File: tooling/generate_instances_types.py (hunk 2 above)
Comment (ferrangrau) [reply tooling/generate_instances_types.py]: I didn't think too much on it, now I generate it manually but it's a good idea automate this...

Comment (ferrangrau) [reply tooling/generate_instances_types.py]: @joaoqalves I will do this in another PR

File: tooling/generate_instances_types.py (hunk 1 above)
Comment (ferrangrau) [reply tooling/generate_instances_types.py]: @joaoqalves done!

File: tooling/generate_instances_types.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import boto3
+import argparse
+
+
+parser = argparse.ArgumentParser(description='Generate a aws instance type enum.')
+
+parser.add_argument('-r', '--region', help="AWS region, now only works with us-east-1", type=str, default="us-east-1")
```
Comment (joaoqalves) [inline tooling/generate_instances_types.py]: perhaps we should read from environ variables that are available to the AWS CLI

Comment (ferrangrau) [reply tooling/generate_instances_types.py]: @joaoqalves I can set it as a non required parameters and take it from the environment variables if they are not set.

Comment (ferrangrau) [reply tooling/generate_instances_types.py]: done @joaoqalves

File: tooling/generate_instances_types.py (hunk 2 above)
Comment (joaoqalves) [reply tooling/generate_instances_types.py]: Ok. I'm not sure if this is a good idea as it is. I'd rather use meta-programming to create classes on the fly or Jinja2 templating

Comment (ferrangrau) [reply tooling/generate_instances_types.py]: @joaoqalves as you say sure it's better jinja2. I will remove this of this PR and create another one to do it better. I think are two different things. Sorry for mix it.

**Review by joaoqalves — APPROVED** [review]

**Pull request merged**